RELEASE-NOTES.md

```diff
@@ -3,6 +3,11 @@
 ## Version 3.1 (dev)
 
 * Added `DerivedPropertyValueSnak`
+* Deprecated `Entity::copy`
+* Deprecated `Entity::getFingerprint`
+* Deprecated `Entity::setFingerprint`
+* Deprecated `Entity::isEmpty`
+* Deprecated `Entity::clear`
```

Comment:
Please add what to use instead, if possible. For example "use `new Item` and `new Property` instead".